PostProcessing/IntegralsPlot.py

```diff
@@ -1,140 +1,64 @@
-    plt.savefig(os.path.join(path,  varname), format = config["Output"].get("fileform", "png").lower(), dpi = config["Output"].getint("dpi", 600))
-
-def setup_figure(): 
-    #setup the figure
-    #in its own function for future modularity
-    plt.figure()
-    plt.tight_layout()
-
-
-def make_plot(data_obj, plot_variable, plotbounds = None, linestyle = '-' ):
-    #plot a single variable
-
-    #get the scaling
-    plotscaling = config["PlotConfig"].get("xscaling", "linear") + config["PlotConfig"].get("yscaling", "linear")
-
-    #get the plot function as determined by the scaling and plot it
-    select_plot_func(plotscaling)(data_obj["time"], data_obj[plot_variable], linestyle = linestyle, label=plot_variable)
-
-    #set the labels
-    plt.xlabel(config["PlotConfig"].get("xlabel", "Time"))
-    plt.ylabel(plot_variable)
-
-    #if the plot bounds are specified, set them
-    if plotbounds is not None:
-        plt.xlim(plotbounds[0], plotbounds[1])
-
-
-def make_plots(data_obj, plot_variables, plotbounds = None,  linestyles = None):
-    #make multiple plots
-
-    #flag to tell if the plots should all be in one figure
-    oneplot = config["VariableData"].getboolean("one_plot", False)
-    if not oneplot:
-
-        #iterate over the variables and plot them
-        for i in range(len(plot_variables)):
-            setup_figure()
-            plotvar = plot_variables[i]
-
-            if linestyles is not None:
-                linestyle = linestyles[i]
-            else:
-                linestyle = '-'
-
-            make_plot(data_obj, plotvar, plotbounds, linestyle)
-            plt.legend()
-            save_fig(plotpath, plotvar)
-
-    #combine all the plots into one
-    else:
-        setup_figure()
-        for i in range(len(plot_variables)):
-            plotvar = plot_variables[i]
-
-            if linestyles is not None:
-                linestyle = linestyles[i]
-            else:
-                linestyle = '-'
-            make_plot(data_obj, plotvar, plotbounds, linestyle)
-        plt.legend()
-        plt.ylabel("") #remove ylabel for combined plot
-        save_fig(plotpath, "combined")
+from Source.OneD import *
+from Source.Common.Utils import get_config
+
+## Load config file
+config = get_config(sys.argv) #includes error checking
+
+#if the plot and movie paths dont exist, create them
+create_output_dirs(config)
+
+#Instantiate printer function that accepts a verbosity level
+verbPrint = VerbosityPrint(config["Header"].getint("verbosity",0))
+verbPrint("Verbosity: ", verbPrint.verbosity)
 
 
 def main():
+    """
+    Main function to plot the data
+
+    Raises:
+        IOError: Data path not found
+        IOError: Data file not found
+        RuntimeError: No plot variables specified
+        SystemError: No plot variables specified
+        SystemError: Number of x limits does not match number of plot variables
+        SystemError: Number of linestyles does not match number of plot variables
```

Comment:
```suggestion
        RuntimeError: No plot variables specified
        RuntimeError: Number of x limits does not match number of plot variables
        RuntimeError: Number of linestyles does not match number of plot variables
```

Reply:
Thanks, forgot to change the docstrings for that